core/src/tasks/schedule/enqueueRuns.ts

```diff
@@ -38,12 +38,26 @@ export class ScheduleEnqueueRuns extends CLSTask {
 
     const schedules = await Schedule.findAll({ where });
 
+    const enqueuedSchedules: Schedule[] = [];
     for (const schedule of schedules) {
       const shouldRun = await schedule.shouldRun({
         ignoreDeltas,
         runIfNotRecurring,
       });
-      if (shouldRun) await schedule.enqueueRun();
+
+      if (shouldRun) {
+        await schedule.enqueueRun();
+        enqueuedSchedules.push(schedule);
+      }
+    }
+
+    if (getGrouparooRunMode() === "cli:run" && enqueuedSchedules.length > 0) {
```

Comment:
You could log this all the time, regardless of runMode

Reply:
Done in d3052ef, but still changed the log level depending on runMode since it's a normal part of what happens in other run modes, but wanted to make it stand out more on `cli:run`